src/modules/terms/dto/create-facility-term-response.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { RESPONSE } from '@ukef/constants';
+
+export class CreateDealResponse {
+  @ApiProperty({
+    example: RESPONSE.FACILITY_TERM_CREATED,
+  })
+  message: string;
+
+  constructor(message: string) {
+    this.message = message;
+  }
```

Comment:
We decided in TFS that we didn't need the constructor (although some of the older code in TFS still has it)

Reply:
Removed